styles.css

```diff
@@ -1,10 +1,48 @@
 @font-face {
-    font-family: "MulishItalic";
-    src: url("./fonts/MulishItalic.ttf");
+  font-family: "MulishItalic";
+  src: url("./fonts/MulishItalic.ttf");
+}
+
+:root {
+  --coll: 4; /* number of columns with buttons */
+  --row: 6; /* number of rows with buttons */
+  --result_height: 140px; /* height in px of calculator result */
 }
 
 body {
-    margin: 0;
-    padding: 0;
-    font-family: "MulishItalic";
+  margin: 0;
+  padding: 0;
+  font-family: "MulishItalic";
+}
+
+.title {
+  text-align: center;
+}
+
+.calculator {
+  background-color: #ff0000;
```

Comment:
Ideally, you should use vars for colors too. But use generics, e.g. `primaryRed`